moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useSearch.js

```diff
@@ -0,0 +1,62 @@
+import { useState, useMemo } from "react";
+
+/**
+ * Attempts to retrieve a valid graphql search value, for example when searching on an
+ * integer/float field but providing it a string, this function returns the value configured
+ * in the invalidValueDefault field in the search object, or null.
+ * @param {string} column - The name of the column to search
+ * @param {*} value - The value in question
+ * @returns {*} - The value output
+ */
+export const getSearchValue = (column, value, queryConfig) => {
```

Comment:
This function is a straight up copy of an existing function. I think that it will be easier to see where to place helpers like this further down the line.